eng/versioning/version_client.txt

```diff
@@ -54,15 +54,19 @@ com.azure:azure-spring-boot-test-aad;1.0.0;1.0.0
 com.azure:azure-spring-boot-test-core;1.0.0;1.0.0
 com.azure:azure-spring-boot-test-cosmosdb;1.0.0;1.0.0
 com.azure:azure-spring-boot-test-keyvault;1.0.0;1.0.0
-com.microsoft.azure:azure-spring-boot;2.3.2;2.3.3-beta.1
-com.microsoft.azure:azure-spring-boot-starter;2.3.2;2.3.3-beta.1
-com.microsoft.azure:azure-active-directory-spring-boot-starter;2.3.2;2.3.3-beta.1
-com.microsoft.azure:azure-active-directory-b2c-spring-boot-starter;2.3.2;2.3.3-beta.1
-com.microsoft.azure:azure-cosmosdb-spring-boot-starter;2.3.2;2.3.3-beta.1
-com.microsoft.azure:azure-data-gremlin-spring-boot-starter;2.3.2;2.3.3-beta.1
-com.microsoft.azure:azure-keyvault-secrets-spring-boot-starter;2.3.2;2.3.3-beta.1
-com.microsoft.azure:azure-servicebus-jms-spring-boot-starter;2.3.2;2.3.3-beta.1
-com.microsoft.azure:azure-spring-boot-metrics-starter;2.3.2;2.3.3-beta.1
+com.microsoft.azure:azure-active-directory-b2c-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
+com.microsoft.azure:azure-active-directory-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
+com.microsoft.azure:azure-cosmosdb-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
+com.microsoft.azure:azure-data-gremlin-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
+com.microsoft.azure:azure-keyvault-secrets-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
+com.microsoft.azure:azure-servicebus-jms-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
+com.microsoft.azure:azure-spring-boot-metrics-starter;2.3.3-beta.1;2.3.3-beta.1
+com.microsoft.azure:azure-spring-boot-starter;2.3.3-beta.1;2.3.3-beta.1
+com.microsoft.azure:azure-spring-boot;2.3.3-beta.1;2.3.3-beta.1
+com.microsoft.azure:spring-cloud-azure-appconfiguration-config-web;1.2.8-beta.1;1.2.8-beta.1
```

Comment:
All of these artifactIds should be starting with azure-*. For example,
spring-cloud-azure-appconfiguration-config should probably be azure-spring-cloud-appconfiguration-config though it's unclear if we need '-cloud" in there. The subdirectories should also be matching the naming convention.

Reply:
I'll change the folder name to `azure-spring-cloud-xxx`.
The artifact name will be changed in this issue: https://github.com/Azure/azure-sdk-for-java/issues/11374

Reply:
Done.

Reply:
If we're freshly adding these to the repository, why wait to set the artifactIds correctly in another issue and not just do it here?